### PR TITLE
mkdir: add page

### PR DESCRIPTION
**Pull request by bbshash**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Comment by CleanMachine1** [comment]
Unfortunately this doesn't fit what TLDR's style is.

There  are already pages for each of the commands since a page is dedicated for just one command.

**Comment by EmilySeville7cfg** [comment]
@bbshash, you can commit smth like that to [devhints](https://devhints.io/) project instead I think because they  accept pages with multiple command's descriptions.

Or you can update existing [mkdir](https://github.com/tldr-pages/tldr/blob/main/pages/common/mkdir.md) or [rmdir](https://github.com/tldr-pages/tldr/blob/main/pages/common/rmdir.md) pages.

**Comment by CleanMachine1** [comment]
@bbshash whilst your contribution is appreciated greatly, it unfortunately doesn't fit the scope or syntax of TLDR.

TLDR-pages are dedicated one command. For example `tldr git-commit` only has git-commit examples 

I am going to close this PR however please feel free to contribute any future pages.
